### PR TITLE
Add pid flag to serve and don't set pid by default

### PR DESCRIPTION
**Pull request by Bwko**
Solves the pid bug of #635

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
Otherwise LGTM.

File: cmd/serve.go [hunk 1]
```diff
@@ -43,6 +43,10 @@ var CmdServ = cli.Command{
 			Name:  "config, c",
 			Value: "custom/conf/app.ini",
 			Usage: "Custom configuration file path",
+		}, cli.StringFlag{
+			Name:  "pid, P",
+			Value: "/var/run/gitea.pid",
+			Usage: "Custom pid file path",
 		},
```
Comment (appleboy) [inline cmd/serve.go]: `git serve` don't need `pid` argument ?

**Review by appleboy — CHANGES_REQUESTED** [review]

**Review by appleboy — APPROVED** [review]

File: cmd/web.go [hunk 2]
```diff
@@ -62,7 +62,7 @@ and it takes care of all the other things for you`,
 		},
 		cli.StringFlag{
 			Name:  "pid, P",
-			Value: "custom/run/app.pid",
+			Value: "/var/run/gitea.pid",
```
Comment (tboerger) [inline cmd/web.go:65]: Why changing the current default?

Comment (Bwko) [reply cmd/web.go:65]: `/var/run/` [Is the normal location for pid files](http://stackoverflow.com/a/5174433)

**Comment by lunny** [comment]
LGTM

**Pull request merged**